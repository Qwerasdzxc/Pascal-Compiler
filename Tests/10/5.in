14
-127 -799 -996 386 982 -486 455 -526 -780 -722 -101 410 709 -248

