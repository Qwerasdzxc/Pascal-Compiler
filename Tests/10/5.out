-996 386 982 -486 -526 -780 -722 410 -248 
-127 -799 455 -101 709 